README.md

```diff
@@ -224,7 +224,23 @@ Wrapping schemas into `m/schema` makes them first class entities. Here `User` is
 ; => true
 ```
 
-## Links
+## Motivation
+
+We are building dynamic multi-tenant systems, where data-models should be first-class: they should drive the runtime value transformations, forms and processes. We should be able to edit the models at runtime, persistent and load back them from database and over the wire, for both Clojure and ClojureScript. Think of [JSON Schema](https://json-schema.org/), but for Clojure/Script.
```

Comment:
```suggestion
We are building dynamic multi-tenant systems where data-models should be first-class: they should drive the runtime value transformations, forms and processes. We should be able to edit the models at runtime, persist them and load them back from database and over the wire, for both Clojure and ClojureScript. Think of [JSON Schema](https://json-schema.org/), but for Clojure/Script.
```

Did not add it to the suggestion, but maybe it's more like *JSON Schema for EDN* than for Clojure/Script.